resources/sdk/clisdkclient/templates/README.mustache

```diff
@@ -0,0 +1,119 @@
+---
+title: Platform API CLI SDK
+---
```

Comment:
add `ispreview: true` to the frontmatter to add the standard "pre-release stuff can change" message to the page when it's rendered in the dev center. The statement below is fine to keep since this file is used in the github repo as well.

Reply:
I am good